File: 5kyu_ROT13.js
function rot13(str) {
  return str.split('').map(function(chr) {
    let num = chr.charCodeAt(0)
    if (((num < 110) && (num > 96)) ||  ((num < 78) && (num > 64))) {
      num += 13
      return String.fromCharCode(num)
    } else if (((num < 123) && (num > 109)) ||  ((num < 91) && (num > 77))) {
      num -= 13
      return String.fromCharCode(num)
    } else {
      return chr
    }
  }).join('')
}

// https://www.codewars.com/kata/52223df9e8f98c7aa7000062

File: 5kyu_Rot13.cs
using System;
using System.Linq;
using System.Collections.Generic;

public class Kata
{
  public static string Rot13(string message)
  {
    var messageAsList = message.ToCharArray();
    for (var i = 0; i < messageAsList.Count(); i++) 
    {
      if (Char.IsLetter(messageAsList[i])) 
      {
        var asciiValOfLetter = (int) messageAsList[i];
        if (asciiValOfLetter < 91) 
        {
          messageAsList[i] = ((asciiValOfLetter - 64 + 13) % 26) == 0 ? (char) 90 : (char) (((asciiValOfLetter - 64 + 13) % 26) + 64);
        }
        else 
        {
          messageAsList[i] = ((asciiValOfLetter - 96 + 13) % 26) == 0 ? (char) 122 : (char) (((asciiValOfLetter - 96 + 13) % 26) + 96);
        }
      }
    }
    return String.Join("", messageAsList);
  }
}

// https://www.codewars.com/kata/530e15517bc88ac656000716
